browser/ui/views/playlist/playlist_edit_bubble_view.cc

```diff
@@ -114,13 +114,8 @@ void PlaylistEditBubbleView::ResetChildViews() {
   RemoveAllChildViews();
 
   constexpr int kIconSize = 16;
-  // TODO(sko) There was feedback that "Added to Play Later" is pretty
-  // confusing. For now we show "Added to Playlist" for clarity. When we
-  // come to the conclusion, revert this to use
-  // PlaylistTabHelper::GetSavedFolderName() if it's needed
   AddChildView(std::make_unique<Row>(
-      l10n_util::GetStringFUTF16(IDS_PLAYLIST_ADDED_TO_PLAYLIST_FOLDER,
```

Comment:
Can we remove "IDS_PLAYLIST_ADDED_TO_PLAYLIST_FOLDER" from the grd too?

Reply:
Removed.